### DISCUSSION TITLE
CreatePdf supported version(s)

### DISCUSSION THREAD

**Discussion by wstaelens** [mono/SkiaSharp#2586] [Q&A]
Documentation on SkiaSharp or Skia does not specify which version(s) of PDF is being supported (e.g. 1.4, 1.5, 1.6, 1.7, ...)

**Which PDF versions are supported?**

(Please also update the documentation)

**Comment by Sebazzz** [comment]
I asked a similar question at QuestPDF: https://github.com/QuestPDF/QuestPDF/discussions/633

It appears it is PDF 1.4 only.

**Comment by mattleibow** [comment]
You can chat the skia team at https://groups.google.com/g/skia-discuss

I am not sure the plans for what they are intending to support and what the interest level is.
